core/language/en-GB.tid

```diff
@@ -2,6 +2,8 @@ title: $:/languages/en-GB
 name: en-GB
 description: English (British)
 author: JeremyRuston
-core-version: >=5.0.0"
+core-version: >=5.0.0
+plugin-type: language
+type: application/json"
```

Comment:
the last 2 elements are needed, otherwise the "pluginswitcher.js" doesn't activate addons for en-GB. .. Those dependencies are hardcoded atm